db/migrate/20210316202859_add_service_name_to_active_storage_blobs.active_storage.rb

```diff
@@ -0,0 +1,18 @@
+# This migration comes from active_storage (originally 20190112182829)
+class AddServiceNameToActiveStorageBlobs < ActiveRecord::Migration[6.0]
+  def up
+    unless column_exists?(:active_storage_blobs, :service_name)
```

Comment:
What's the `service_name` column for?

Reply:
If we were using AWS, or some other third party file storage I believe the name of the service used to store that particular blob will be added to that column. Not sure what ours will contain. Maybe blank, maybe "filestore".

Reply:
Ahh okay, move along then.

Reply:
Why do you need a guard against the column existing?